Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
> I believe the parameter of deep = true in this case already triggers an error if the app is using http-foundation <=2.8. 

If you can test that and make sure the code is already bad for sf < 2.8, by all means solve this globally. A unit test will help a lot with proving that, because we run tests for different versions of Symfony on Travis.